3634/2/helpers/event_handler.rb

```diff
@@ -0,0 +1,62 @@
+require 'fileutils'
+require_relative 'save_files'
+
+module EventHandler
+  CAMPLATITUDE = (53.913264..53.917233).freeze
```

Comment:
CAMP_LATITUDE, try to use more human naming

Reply:
Done